examples/startup_namer/lib/main.dart

```diff
@@ -0,0 +1,109 @@
+import 'package:flutter/material.dart';
+import 'package:english_words/english_words.dart';
+
+void main() => runApp(new MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+      title: 'Startup Name Generator',
+      theme: new ThemeData(
+        primaryColor: Colors.white,
+      ),
+      home: new RandomWords(),
+    );
+  }
+}
+
+class RandomWords extends StatefulWidget {
+  @override
+  createState() => new RandomWordsState();
+}
+
+class RandomWordsState extends State<RandomWords> {
+  // Dart lets you define lists in different ways
+  final _suggestions = <WordPair>[];  
+  final List<WordPair> _saved = [];
+
+  final TextStyle _biggerFont = new TextStyle(fontSize: 18.0);
+
+  @override
+  Widget build(BuildContext context) {
+    return new Scaffold(
+      appBar: new AppBar(
+        title: new Text('Startup Name Generator'),
+        actions: <Widget>[
+          new IconButton(
+            icon: new Icon(Icons.list),
+            onPressed: _pushSaved,
+          ),
+        ],
+      ),
+      body: _buildSuggestions(),
+    );
+  }
+
+  Widget _buildRow(WordPair pair) {
+    final alreadySaved = _saved.contains(pair);
```

Comment:
this is O(N), which will be expensive if they save a lot of pairs. Generally I'd use a Set here, but you want to show them later so maybe a Set and a List? Or just assume they won't save many, and that O(N) is fine.

Reply:
I originally had `_saved` as `Set` but changed it to `List` in order to not confuse people with new concepts. But now that I think of it, I think it will be okay in the flow of the tutorial (Matt has the tutorial start with only the suggestions and only after that adds the saved list).

So I'll change this back to `Set`.

// cc @mjohnsullivan